styles/documentation/footer.css

```diff
@@ -1,27 +1,33 @@
 .footer {
-  margin: 2rem 3rem;
+  /* margin: 2rem 3rem; */
```

Comment:
suggestion - you can remove this if not required

Reply:
removed